vxdetector/tests/test_files_manager.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python
+
+import unittest
+import os
+import tempfile
+import shutil
+import vxdetector.files_manager as fm
+
+
+class test_tmp_dir(unittest.TestCase):
+    def test_basic_function(self):
+        path = os.path.dirname(__file__)
+        temp_path = fm.tmp_dir(path, temp_path='')
+        self.assertTrue(os.path.exists(temp_path))
+        fm.tmp_dir(path, temp_path)
+        self.assertFalse(os.path.exists(temp_path))
+
+
+class test_get_lib(unittest.TestCase):
+    def setUp(self):
+        self.fp_tmpdir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        shutil.rmtree(self.fp_tmpdir)
+
+    def test_program_path(self):
+        program_path = fm.get_lib()
+        self.assertTrue(os.path.exists(program_path))
+        self.assertTrue(os.path.exists(f'{program_path}/vxdetector/'))
+        self.assertTrue(os.path.exists(f'{program_path}/vxdetector/'
+                                       'VXdetector.py'))
```

Comment:
I don't think it is necessary to test if source files are present. That's the obligation of the user / installer.

Reply:
This is more to test if files_manager gets the correct 'progam_path'
But considering files_manager gets the 'program_path' via 
program_path = os.path.dirname(__file__)
it might be unnecessary.